config/filter.d/nginx-http-auth.conf

```diff
@@ -3,15 +3,23 @@
 
 [Definition]
 
+extendregex = 
 
 failregex = ^ \[error\] \d+#\d+: \*\d+ user "\S+":? (password mismatch|was not found in ".*"), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
+            %(extendregex)s
 
 ignoreregex = 
 
+protect_web_apps = ^ \[error\] \d+#\d+: \*\d+ user \S* ?(not found|not authorized|password mismatch|sent invalid token|misbehaved): .+, client: <HOST>
```

Comment:
Certainly the same issue as the one you noticed with lighttpd.
Do you want me to append the following to this new rule, as in line 8 ?
`, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$`